govuk-prototype-kit.config.json

```diff
@@ -4,7 +4,10 @@
     "/sass/_step-by-step-nav.scss",
     "/sass/_step-by-step-related.scss"
   ],
+  "assets": [
+    "/dist/main.js.map"
+  ],
   "scripts": [
-    "/javascripts/step-by-step-nav.js"
+    "/dist/main.js"
```

Comment:
I don't think we can change the name, as old versions of the template included the script in the `head`, meaning if there isn't a file at `public/javascripts/step-by-step-nav.js` they will get an error in the terminal.

Reply:
Now that we're using an extension for this we have to use a URL under `/extension-assets`.  This means that users will need to remove references to the old javascript.  If they don't the step-by-step will still work but it will include the error in the console.

Reply:
Ah, got it, thanks!